config/development.json5

```diff
@@ -1,4 +1,17 @@
 {
+  uiUrl: 'http://localhost:3000/offer?peerId=<<peerId>>',
```

Comment:
This will depend on Marketplace UI, will be updated once the integration gets done.